src/app/static/src/js/components/runWorkflow/workflowSummary/workflowSummaryReports.vue

```diff
@@ -22,7 +22,26 @@
                                     <div class="workflow-summary-text">
                                         <span class="text-muted d-inline-block">Parameters</span>
                                         <div v-if="hasParams(report)">
-                                            <p id="params" v-for="(value, key, index) in report.params" :key="index">{{ key }}: {{ value }}</p>
+                                            <p id="params"
+                                               v-for="(value, key, index) in report.params"
+                                               :key="index">{{ key }}: {{ value }}</p>
+                                            <div :id="`default-params-${index}`">
+                                                <a :href="`#collapseSummary-${index}`"
+                                                   class="pt-2 d-inline-block small"
+                                                   data-toggle="collapse"
+                                                   aria-expanded="false"
+                                                   aria-controls="collapseSummary"
+                                                >Show default...</a>
```

Comment:
This text should probably be 'Show defaults..' or 'Show default values...' and should change to 'Hide defaults..' when expanded.